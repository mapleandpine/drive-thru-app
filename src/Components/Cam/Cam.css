.Cam {
  background-color: #555;
  color: #fff;
  grid-column: col;
  grid-row: row;
  padding: 10px;
  padding-bottom: 0;
  position: relative;
}

.Cam::before {
  background-image: linear-gradient(22.5deg, #000 25%, transparent 25%, transparent 50%, #000 50%, #000 75%, transparent 75%, transparent);
  background-size: 4px 4px;
  bottom:0;
  content:'';
  left:0;
  position:absolute;
  right:0;
  top:0;
  z-index:-1;
}

.Cam::after {
  bottom: 12px;
  content: "Order Cam";
  display: block;
  left: 18px;
  position: absolute;
}

.CamVideo {
  border-radius: 8px;
  filter: blur(1px) brightness(.9) contrast(1.1) grayscale(.75) opacity(.65) saturate(1.8) sepia(.25);
  width: 100%;
}
