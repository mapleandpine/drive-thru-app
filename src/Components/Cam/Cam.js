import React, { Component } from 'react';
import image from './Cam.png';
import video from './Cam.mp4';
import './Cam.css';

class Cam extends Component {

  render() {
    return (
      <div className="Cam">
        <video className="CamVideo" autoPlay loop poster={image}>
           <source src={video} type="video/mp4"/>
        </video>
      </div>
    );
  }
}

export default Cam;
