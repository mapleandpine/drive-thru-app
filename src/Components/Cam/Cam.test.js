import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Cam from './Cam';

describe(Cam, () => {
  it('renders and matches our snapshot', () => {
    const component = renderer.create(
      <Cam />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
