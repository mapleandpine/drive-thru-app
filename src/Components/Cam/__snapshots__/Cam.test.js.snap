// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class Cam extends _react.Component {

  render() {
    return (
      _react2.default.createElement('div', { className: 'Cam', __source: { fileName: _jsxFileName, lineNumber: 10 }, __self: this },
        _react2.default.createElement('video', { className: 'CamVideo', autoPlay: true, loop: true, poster: _Cam2.default, __source: { fileName: _jsxFileName, lineNumber: 11 }, __self: this },
          _react2.default.createElement('source', { src: _Cam4.default, type: 'video/mp4', __source: { fileName: _jsxFileName, lineNumber: 12 }, __self: this }))));



  }} renders and matches our snapshot 1`] = `
<div
  className="Cam"
>
  <video
    autoPlay={true}
    className="CamVideo"
    loop={true}
    poster="Cam.png"
  >
    <source
      src="Cam.mp4"
      type="video/mp4"
    />
  </video>
</div>
`;
